docker/compose/docker-compose-nfs.yml

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+version: '2'
+volumes:
+  nfs:
+    driver: local
+    driver_opts:
+      type: nfs
+      o: addr=172.16.39.2,rw,sync
```

Comment:
This is just a sample to load externally deployed NFS. Could not figure out how to pass is via a parameter to docker-compose.